Hardware/Tessel_2_Overview.md

```diff
@@ -0,0 +1,84 @@
+# Hardware Overview of Tessel 2
+
+## Quick specs
+
+At a glance, the main hardware components of Tessel 2 are:
+
+*  580MHz WiFi router system on chip ([Mediatek MT7620n](http://www.anz.ru/files/mediatek/MT7620_Datasheet.pdf)) running linux ([OpenWRT](https://openwrt.org/))
+* 64 MB of DDR2 RAM
+* 32 MB of flash storage
+* 2 High-speed USB 2.0 ports
+* micro USB port
+* 10/100 Ethernet port (RJ-45 jack)
+* 48MHz ARM Cortex M0 microcontroller ([Atmel SAMD21](http://www.atmel.com/Images/Atmel-42181-SAM-D21_Datasheet.pdf))
+* Two module ports that are much more capable than their predecessors
+* 4 LEDs
+* Reset button
+
+Above-and-beyond features:
+
+* Router-grade 802.11b/g/n WiFi, including access point mode ([Tessel can be a router](http://tessel.github.io/t2-start/ap.html))
+* 16 GPIO broken out as a pair of multi-purpose module ports
+* Individual control over and protection for all outward-facing power buses (USB and module ports)
+* A form factor designed for abstraction and flexibility in the hardware, software, and mechanical worlds as you scale from prototype to production
+
+## Tessel 2 Layout
+
+On the board, Tessel 2 is divided into three functional blocks:
+
+![](http://67.media.tumblr.com/83018b7f2595c1f4b3877f74eb86d134/tumblr_inline_nkz56hKzbm1s75tgz.png)
+
+These can be abstracted as follows:
+
+![](http://65.media.tumblr.com/559c4d45074644a5c335987bec7c698e/tumblr_inline_nkz56bL5SC1s75tgz.png)
+
+### Abstraction boundaries of Tessel 2
+
+The board employs a processor/coprocessor architecture. The Mediatek runs your user code, hosts USB devices, handles the network connections (be they wired, wireless, or cellular over USB), and communicates with the SAMD21.
+
+The SAM acts as a coprocessor and handles real-time, low-level IO through the module ports, USB comms through the Micro USB port, and programming the device as a whole.
+
+The two chips are connected by a SPI bridge that also includes the onboard flash (the [readme for Tessel 2’s firmware repo goes into more detail here](https://github.com/tessel/v2-firmware/blob/master/README.md)).
+
+The whole system is powered from the single Micro USB (device) port.
```

Comment:
Tessel could also be powered by the headers with the 5Vin next to the microUSB port, correct? 

Also (picky), "**micro** USB" is used earlier and "**Micro** USB" is used here. It would be nice to standardize on what it is called.